fast64_internal/sm64/sm64_geolayout_writer.py

```diff
@@ -3333,21 +3240,12 @@ def sm64_geo_writer_register():
 
     bpy.types.Scene.textDumpGeo = bpy.props.BoolProperty(name="Dump geolayout as text", default=False)
     bpy.types.Scene.textDumpGeoPath = bpy.props.StringProperty(name="Text Dump Path", subtype="FILE_PATH")
-    bpy.types.Scene.geoExportPath = bpy.props.StringProperty(name="Directory", subtype="FILE_PATH")
     bpy.types.Scene.geoUseBank0 = bpy.props.BoolProperty(name="Use Bank 0")
     bpy.types.Scene.geoRAMAddr = bpy.props.StringProperty(name="RAM Address", default="80000000")
     bpy.types.Scene.geoTexDir = bpy.props.StringProperty(name="Include Path", default="actors/mario/")
     bpy.types.Scene.geoSeparateTextureDef = bpy.props.BoolProperty(name="Save texture.inc.c separately")
     bpy.types.Scene.geoInsertableBinaryPath = bpy.props.StringProperty(name="Filepath", subtype="FILE_PATH")
     bpy.types.Scene.geoIsSegPtr = bpy.props.BoolProperty(name="Is Segmented Address")
-    bpy.types.Scene.geoName = bpy.props.StringProperty(name="Directory Name", default="mario")
-    bpy.types.Scene.geoGroupName = bpy.props.StringProperty(name="Name", default="group0")
-    bpy.types.Scene.geoExportHeaderType = bpy.props.EnumProperty(
-        name="Header Export", items=enumExportHeaderType, default="Actor"
-    )
-    bpy.types.Scene.geoCustomExport = bpy.props.BoolProperty(name="Custom Export Path")
-    bpy.types.Scene.geoLevelName = bpy.props.StringProperty(name="Level", default="bob")
-    bpy.types.Scene.geoLevelOption = bpy.props.EnumProperty(items=enumLevelNames, name="Level", default="bob")
```

Comment:
Since you are removing these, you need some upgrade code.

Reply:
afaik they're unused so the props will just be vestigial remnants of old classes, and since they aren't called they won't error out